microsite/src/main/resources/microsite/css/override.css

```diff
@@ -42,29 +43,33 @@
     background: url("../img/sidebar_brand.png") no-repeat;
 }
 
-@media (min--moz-device-pixel-ratio: 1.3), (-o-min-device-pixel-ratio: 2.6 / 2), (-webkit-min-device-pixel-ratio: 1.3), (min-device-pixel-ratio: 1.3), (min-resolution: 1.3dppx) {
+ @media (min--moz-device-pixel-ratio: 1.3), (-o-min-device-pixel-ratio: 2.6 / 2), (-webkit-min-device-pixel-ratio: 1.3), (min-device-pixel-ratio: 1.3), (min-resolution: 1.3dppx) {
```

Comment:
Can you include some screenshots of all this? Would be helpful!

Reply:
![screencapture-127-0-0-1-4000-scalaz-zio-2019-03-13-15_25_53](https://user-images.githubusercontent.com/3358827/54330541-df403100-461e-11e9-9d4a-f1959df2db0f.png)